app/forms/hub/create_client_form.rb

```diff
@@ -61,6 +61,27 @@ def save(current_user)
         tax_returns_attributes: @tax_returns_attributes.map { |_, v| create_tax_return_for_year?(v[:year]) ? tax_return_defaults.merge(v) : nil }.compact
       )
 
+      locale = @client.intake.preferred_interview_language == "es" ? "es" : "en"
```

Comment:
Wasn't sure the best way to set the locale on these messages since they depend on the preferred_interview_language

Reply:
that makes sense to me.